testground/README.md

```diff
@@ -1,13 +1,20 @@
-# Guide
-
 [Testground documentation](https://docs.testground.ai/)
 
-## Getting started
+## Build Image
+
+>  Prerequisites: nix, linux remote builder for macOS
+
+You can test with the prebuilt images in [github registry](https://github.com/crypto-org-chain/cronos/pkgs/container/cronos-testground), or build the image locally:
 
-### Prerequisites
+```bash
+$ nix build .#testground-image
+# for mac: nix build .#legacyPackages.aarch64-linux.testground-image
```

Comment:
do we need link https://nixos.org/manual/nixpkgs/unstable/#sec-darwin-builder

Reply:
that's one of the ways to run a remote builder.